lms/static/js/commerce/views/receipt_view.js

```diff
@@ -61,6 +63,18 @@ var edx = edx || {};
                 $courseNamePlaceholder.text(responseData.name);
             });
         },
+        renderUserFullNamePlaceholder: function (username) {
+            var userModel = Backbone.Model.extend({
```

Comment:
@AlasdairSwan I have added backbone model here. However I tried to put this model into separate file but was not able to import here. Is there any way to import in this [method](https://github.com/edx/edx-platform/blob/master/lms/static/js/commerce/views/receipt_view.js#L309) or is it ok to leave it here now because eventually we are going to refactor this page.